scripts/install_nfs_mounts.sh

```diff
@@ -6,7 +6,13 @@ case "$OSTYPE" in
         mount_dir="/c/Users"
         ;;
     linux*)
-        export_dir="/home"
+        if [ -d "/srv/nfs/home" ]; then
+            export_dir="/srv/nfs/home"
+        elif [ -d "/srv/home" ]; then
+            export_dir="/srv/home"
+        else
+            export_dir="/home"
+        fi
```

Comment:
@ptramsey I think your input would be valuable here, this seems fine to me, but I am not too familiar with linux dev workflows or what the "standard" is.

@daniel-rollings-fivestars Do you think maybe a small "echo: 'your nfs home export is: <dir>'` would be valuable here? I can see this possibly being confusing if aladdin picks a different dir other than "/home" (even if the others are more secure)

Reply:
Agreed, and though I'm not sure it's enough notification, here's a start.

Reply:
I've found the most significant blockers driving this PR.  Some are alluded to here:

https://confluence.fivestars.com/display/ENG/Running+Aladdin+on+Linux

But most significantly, hard-coded 192.168.99.1 addresses in Aladdin do not match what got set up in Docker.  This needs to be corrected for anyone who's not running in VirtualBox.  Perhaps this script needs to source configuration variables from a file set to current defaults, but the --init can provision correctly detected ones for this use case?